quetz/main.py

```diff
@@ -84,6 +86,8 @@
     https_only=config.session_https_only,
 )
 
+app.add_middleware(PrometheusMiddleware)
```

Comment:
Can we do this the other way around? Have a function in quetz.metrics that is called `init` and takes an `app` as input and installs the metrics routes and middleware?

e.g..

```
from quetz import metrics
...

metrics.init(app)
```

Reply:
Done